en/hosting_guide/configuration.rst

```diff
@@ -0,0 +1,50 @@
+Server Configuration
+====================
+
+To know more about server configuration see `requirements`_.
+
+Cron jobs
+---------
+
+Cron jobs are used for tasks (commands or shell scripts) to run periodically at fixed times, dates, or intervals.
+Cron jobs typically automate system maintenance.
+
+Cron jobs on Open Orchestra
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open Orchestra has 2 cron jobs created with the provisioning:
+
+* **Sitemap**: this task automatically generates sitemap files for every sites.
```

Comment:
to automatically, don't put this task, we already know which task we are talking about